### DISCUSSION TITLE
Non-functional opening window

### DISCUSSION THREAD

**Discussion by rrandor1** [pawelsalawa/sqlitestudio#3749] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.
I double-click on the .db file that I want to open and I basically get a non-functional window. Every single function is grey-ed out except the "Test Connection" and Cancel buttons on the Database window. Even the Quit SQLiteStudio menu option in grey. If I cancel on the Database window then there is nothing that is functional, none of the menu options are available. 

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.
See above.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.
Mac OS 10.15.4

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.
The About is greyed out, non-funcitonal.

**Comment by pawelsalawa** [comment]
1. What's your SQLiteStudio version?
2. Could you make a screenshot of this and put it in the comment?

**Comment by rrandor1** [comment]
Sorry, but I was completely stuck and had to get my work done, so I deleted it thoroughly with MacCleanse and then reinstalled. The version was only a few months old. 

Roger Barr

+52 222 739 8334 (WhatsApp & Cell)
+1 510 480 1070
Skype: rrandor
www.canhealyourself.com


On May 8, 2020 at 17:29:37, Paweł Salawa (notifications@github.com) wrote:

What's your SQLiteStudio version?
Could you make a screenshot of this and put it in the comment?
